MSAL/src/MSALLogger.m

```diff
@@ -59,6 +59,7 @@ - (id)init
     // will most likely be too noisy for most usage.
     self.level = MSALLogLevelInfo;
     self.consoleLogging = YES;
+    self.PiiLoggingEnabled = NO;
```

Comment:
Copying how it's done in .NET *shrug*. I've argued in the past that it's best for BOOL flags to default to false as well.